dev/com.ibm.ws.junit.extensions/src/test/common/zos/exec/README.txt

```diff
@@ -0,0 +1,5 @@
+1. Java Code  : test.common.zos.ZFATNativeHelper.java.
+   Native Code: com.ibm.zos.native/zfat_wlm_services_jni.c.
```

Comment:
this readme and the `test.common.zos` package is referencing files that don't exist in this repo.  Perhaps we could split out the zOS stuff?

Reply:
I'll leave that as a follow on exercise.